core/src/main/scala/org/apache/gearpump/cluster/client/RunningApplication.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gearpump.cluster.client
+
+import akka.pattern.ask
+import akka.actor.ActorRef
+import akka.util.Timeout
+import org.apache.gearpump.cluster.ClientToMaster.{ResolveAppId, ShutdownApplication}
+import org.apache.gearpump.cluster.MasterToClient.{ResolveAppIdResult, ShutdownApplicationResult}
+import org.apache.gearpump.util.ActorUtil
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+class RunningApplication(val appId: Int, master: ActorRef, timeout: Timeout) {
+  import scala.concurrent.ExecutionContext.Implicits.global
+  lazy val appMaster: ActorRef = resolveAppMaster(appId)
+
+  def shutDown(): Unit = {
+    val result = ActorUtil.askActor[ShutdownApplicationResult](master,
+      ShutdownApplication(appId), timeout)
+    result.appId match {
+      case Success(_) =>
+      case Failure(ex) => throw ex
+    }
+  }
+
+  def askMaster[T](msg: Any): Future[T] = {
+    appMaster.ask(msg)(timeout).asInstanceOf[Future[T]]
+  }
+
+//  Todo
```

Comment:
I'm afraid the intention for these lines could be forgotten after a while so maybe better just remove them for now.